cmd/geoipupdate/main.go

```diff
@@ -39,11 +39,11 @@ func main() {
 	}
 
 	config, err := geoipupdate.NewConfig(args.ConfigFile, defaultDatabaseDirectory, args.DatabaseDirectory, args.Verbose)
+	log.Printf("Using config file %s", args.ConfigFile)
```

Comment:
We should only log this when logging verbose. I think Paul's request could be solved by backing out of this change and changing:

```
		fatalLogger("error loading configuration file", err)
```

to:

```
		fatalLogger(fmt.Sprintf("error loading configuration file %s", args.ConfigFile) err)

Reply:
Thank you for the review and suggestion!  I realized my way would start sending emails all the time after Andy pointed out.